scripts/generate_sitemap.py

```diff
@@ -44,6 +47,10 @@ def __init__(self):
                 region_name='us-east-1'
             )
 
+    def __del__(self):
```

Comment:
Using `__del__` is a bit counterintuitive. `del` is typically used to remove an object in memory, so doing IO is unexpected. Maybe rename to `clean()` or `cleanup()`.

That said, is this actually called anywhere?